react/features/transcription/components/TranscriptionSubtitles.web.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+/**
+ * The React {@code Component} class of {@link TranscriptionSubtitles}
+ * for style changes from css/_transcriptionSubtitles
+ */
+const className = 'transcription-subtitles';
+
+/**
+ * React {@code Component} which can display speech-to-text results from
+ * Jigasi as subtitles.
+ *
+ * Jigasi will send a JSON object via
+ * {@code ConferenceEvents.ENDPOINT_MESSAGE_RECEIVED}. An example of a json
+ * object sent by jigasi is:
+ *
+ * {
+ *  'jitsi-meet-muc-msg-topic':'transcription-result',
+ *  'payload':{
+ *     'transcript':[
+ *        {
+ *           'confidence':0,
+ *           'text':'how are'
+ *        }
+ *     ],
+ *     'is_interim':true,
+ *     'language':'en-US',
+ *     'message_id':'8360900e-5fca-4d9c-baf3-6b24206dfbd7',
+ *     'event':'SPEECH',
+ *     'participant':{
+ *        'name':'Nik',
+ *        'id':'2fe3ac1c'
+ *     },
+ *     'stability':0.009999999776482582,
+ *     'timestamp':'2017-08-21T14:35:46.342Z'
+ *  }
+ * }
+ *
+ */
+class TranscriptionSubtitles extends React.Component<Props, State> {
```

Comment:
Props and State was declared because I used `@flow` initially as the file initially used `declare var JitsiMeetJS: Object;` which needed a type declaration.
Ended up removing `@flow` after refactoring. 
Adding it :+1: